internal/config/config.go

```diff
@@ -364,6 +346,12 @@ func Default() *Config {
 				UseMarketplaceImage:   toPtr(false),
 			},
 			OpenStack: &OpenStackConfig{
+				CloudsYAMLPath: func() string {
+					if os.PathSeparator == '/' {
+						return "~/.config/openstack/clouds.yaml"
```

Comment:
Let's keep this empty, because the path is added by `ReadCloudsYAML` and to keep the TF provider variable default consistent with the CLI.